src/components/Explorer/Nucleotide/Intro.jsx

```diff
@@ -4,47 +4,70 @@ import { ExternalLink } from 'common';
-                Search Info
+
+        <div id="info" class="text-left" >
+            <button className="text-xl m-auto" onClick={() => setShowInfo(!showInfo)}>
+                ► Search Information
             </button>
             <div className={showInfo ? "block" : "hidden"}>
                 <div className="my-2 sm:ml-12">{info}</div>
             </div>
         </div>
-        <div id="examples">
-            <button className="text-xl block m-auto" onClick={() => setShowExamples(!showExamples)}>
-                Examples
+
+        <div id="examples" class="text-left" >
+            <button className="text-xl m-auto" onClick={() => setShowExamples(!showExamples)}>
+                ► Examples
             </button>
             <div className={showExamples ? "block" : "hidden"}>
                 <div className="my-2 sm:ml-12">{examples}</div>
             </div>
         </div>
+
+        <div id="mascot" class="text-center" >
+            <button className="text-center m-auto" onClick={() => setShowMascot(!showMascot)}>
+                <div align="center" id="Frank">
+                    <img align="center" className="h-64" src="/Frank_Ginger.png" alt="Frank and Ginger, the Serratus mascots" />
+                    <i>Serratus mascots: Frank and Ginger</i>
+                </div>
+            </button>
+            <div className={showMascot ? "block" : "hidden"}>
+                <div className="my-2 sm:ml-12">{mascot}</div>
+            </div>
+        </div>
+
+
     </>
 }
 
 export default Intro;
 
 const info = <>
+
+    <img className="m-auto h-64" src="/nt_search.png" alt="Nucleotide Search overview" />
+
     <div className="font-bold mb-2">
-        Sequence Reference: <ExternalLink href="https://github.com/ababaian/serratus/wiki/ref_cov3ma" className="text-blue-600">cov3ma</ExternalLink>
+        Sequence Reference: <i>cov3ma</i>
+        <ExternalLink href="https://github.com/ababaian/serratus/wiki/ref_cov3ma" className="text-blue-600"> (wiki) </ExternalLink>
+        <ExternalLink href="https://s3.amazonaws.com/lovelywater/seq/cov3ma/cov3ma.fa" className="text-blue-600"> (cov3ma.fa) </ExternalLink>
     </div>
     <div>
-    A nucleotide pangenome of:
+    Short reads were aligned against a nucleotide pangenome.
     <ol className="list-decimal list-inside">
         <li>GenBank Coronaviridae sequences clustered at 99% identity (n=10,101)</li>
         <li>non-retrovirus Vertebrate RefSeq Viruses (n=2,849)</li>
         <li><ExternalLink href="https://card.mcmaster.ca/" className="text-blue-600">CARD</ExternalLink> Antimicrobial Resistance (AMR) genes clustered at 95%</li>
     </ol>
     </div>
+    <div> Operational sensitivity: 85-99% genome-wide nucleotide identity. </div>
     <div className="font-bold my-2">
         SRA Search
+        <ExternalLink href="https://github.com/ababaian/serratus/wiki/SRA-queries" className="text-blue-600"> (wiki v201210) </ExternalLink>
```

Comment:
`wiki v201210` here but just `wiki` for RdRP?

Reply:
Yes, the nt search has it's own sub-folder based on an older version release for archiving sake.